rest/processing/process.py

```diff
@@ -60,7 +60,7 @@ class Process:
     def __init__(self, process, width=None, height=None, user=User(), user_defined_processes={}, request_type=None):
         self.DEFAULT_EPSG_CODE = 4326
         self.DEFAULT_RESOLUTION = (10, 10)
-        self.MAXIMUM_SYNC_FILESIZE_BYTES = 5000000
+        self.MAXIMUM_SYNC_FILESIZE_BYTES = 100000000
```

Comment:
This is not used anywhere anymore I believe, so it can probably be removed

Reply:
Removed